### ISSUE TITLE
Is there a policy for treatment of records from OMIM using 'with or without'?

### ISSUE THREAD

**Issue by maglott** [monarch-initiative/mondo#4631]
If NCBI reviews recent terms as the phenotypes for allelic variants, such as https://omim.org/allelicVariants/600024
RHIZOMELIC SKELETAL DYSPLASIA WITH PELGER-HUET ANOMALY
we note that the preferred term for https://omim.org/entry/618019 is RHIZOMELIC SKELETAL DYSPLASIA WITH OR WITHOUT PELGER HUET ANOMALY; SKPHA
with PELGER-HUET ANOMALY WITH MILD SKELETAL ANOMALIES; PHASK as an alternate and
Pelger-Huet anomaly with mild skeletal anomalies reported in genemap2.txt.

If a rule is followed that a term labeled as alternate by OMIM is the same as that labeled as preferred, then the more specific term (only with) would be equated with the more general (with or without).

Is that what we all should do, or should specificity be retained?
In other words, are there 3 records, 2, or 1?
RHIZOMELIC SKELETAL DYSPLASIA WITH OR WITHOUT PELGER HUET ANOMALY
RHIZOMELIC SKELETAL DYSPLASIA WITH PELGER-HUET ANOMALY
Pelger-Huet anomaly with mild skeletal anomalies

**Comment by nicolevasilevsky** [comment]
@maglott we'll take a look at this asap, thanks for reporting.

**Comment by nicolevasilevsky** [comment]
We don't have a specific policy for this - this would be handled on a case-by-case basis. In general, I think we'd follow OMIM and keep the terms lumped together. If another resource like Orphanet chooses to make a distinction, this would show up in Mondo.

If you have a use case where you need the terms to be split, please let us know and we can do that.

**Comment by maglott** [comment]
Thanks, Nicole.  NCBI will retain the specific value for the cases where OMIM names the phenotype of an allele with the specific.
